bookkeeper-server/src/main/java/org/apache/bookkeeper/client/api/LedgerEntries.java

```diff
@@ -35,24 +35,11 @@
     LedgerEntry getEntry(long entryId);
 
     /**
-     * In this method, It does not increment the reference counts of ByteBuf for the entries in this LedgerEntries.
+     * This method does not increment the reference count of ByteBuf for the entries in this LedgerEntries.
```

Comment:
Calling this methods does not modify the reference count of the ByteBuf in the returned LedgerEntry objects.
The caller who calls {@link #iterator()} should make sure that they do not call release on the LedgerEntry objects to avoid a double free. All reference counts will be decremented when the containing LedgerEntries object is closed.

Reply:
will change it.